sdk/typescript/src/transactions/TransactionBlockData.ts

```diff
@@ -136,6 +137,10 @@ export class TransactionBlockDataBuilder {
 		return transactionData;
 	}
 
+	static restoreFromV1(data: SerializedTransactionDataBuilderV1) {
+		return this.restore(data);
```

Comment:
this is just a placeholder that confirms the new valibot types are compatible with the superstruct types